centreon/www/front_src/src/Resources/Actions/Visualization/index.tsx

```diff
@@ -0,0 +1,75 @@
+import { makeStyles } from 'tss-react/mui';
+import { useTranslation } from 'react-i18next';
+
+import { Box, Typography } from '@mui/material';
+
+import viewByServiceActive from '../../icons/view_service_actif.svg';
+import viewByServiceInactive from '../../icons/view_service_inactif.svg';
+import viewByHostActive from '../../icons/view_host_actif.svg';
+import viewByHostInactive from '../../icons/view_host_inactif.svg';
+import viewByAllInactive from '../../icons/view_all_inactif.svg';
+import viewByAllActive from '../../icons/view_all_actif.svg';
+import {
+  labelDisplayView,
+  labelViewByHost,
+  labelViewByService,
+  labelAll
+} from '../../translatedLabels';
+import { Visualization } from '../../models';
+
+import Action from './Action';
+
+const useStyles = makeStyles()((theme) => ({
+  container: {
+    alignItems: 'center',
+    display: 'flex',
+    gap: theme.spacing(0.5)
+  },
+  text: {
+    marginRight: theme.spacing(1.5)
+  }
+}));
+
+const actions = [
+  {
+    IconOnActive: viewByAllActive,
+    IconOnInactive: viewByAllInactive,
+    title: labelAll,
+    type: Visualization.All
+  },
+  {
+    IconOnActive: viewByHostActive,
+    IconOnInactive: viewByHostInactive,
+    title: labelViewByHost,
+    type: Visualization.Host
+  },
+  {
+    IconOnActive: viewByServiceActive,
+    IconOnInactive: viewByServiceInactive,
+    title: labelViewByService,
+    type: Visualization.SERVICE
+  }
+];
+const View = (): JSX.Element => {
```

Comment:
this is a bit confusing, we now have viewMode, visualizationActions and view. Can we try to have name that match the underlying concepts a bit better?

Reply:
my suggestion is to keep viewMode, Change view, and Visualization to visualizationActions. what do you think?

Reply:
what about listingVariant instead of view mode? that's how the corresponding type is called

Reply:
should we rename View to VisualizationActions as well?